openbb_platform/providers/oecd/openbb_oecd/utils/constants.py

```diff
@@ -47,6 +47,126 @@
 
 CODE_TO_COUNTRY_GDP = {v: k for k, v in COUNTRY_TO_CODE_GDP.items()}
 
+unemployment_list = [
+    "australia",
+    "austria",
+    "belgium",
+    "canada",
+    "czech_republic",
+    "denmark",
+    "finland",
+    "france",
+    "germany",
+    "greece",
+    "hungary",
+    "iceland",
+    "ireland",
+    "italy",
+    "japan",
+    "korea",
+    "luxembourg",
+    "mexico",
+    "netherlands",
+    "norway",
+    "poland",
+    "portugal",
+    "slovak_republic",
+    "spain",
+    "sweden",
+    "turkey",
+    "united_kingdom",
+    "united_states",
+    "chile",
+    "estonia",
+    "israel",
+    "slovenia",
+    "oecd_total",
+    "G7",
+    "euro_area_19",
+    "latvia",
+    "lithuania",
+    "colombia",
+    "european_union_27",
+    "costa_rica",
+]
+COUNTRY_TO_CODE_UNEMPLOYMENT = {
+    k: v for k, v in COUNTRY_TO_CODE_GDP.items() if k in unemployment_list
+}
+CODE_TO_COUNTRY_UNEMPLOYMENT = {v: k for k, v in COUNTRY_TO_CODE_UNEMPLOYMENT.items()}
+
+ppi_list = [
+    "AUT",
+    "BEL",
+    "CAN",
+    "CZE",
+    "DNK",
+    "FIN",
+    "FRA",
+    "DEU",
+    "GRC",
+    "HUN",
+    "IRL",
+    "ITA",
+    "LUX",
+    "NLD",
+    "NOR",
+    "POL",
+    "PRT",
+    "SVK",
+    "ESP",
+    "SWE",
+    "CHE",
+    "GBR",
+    "USA",
+    "CHL",
+    "COL",
+    "EST",
+    "LVA",
+    "LTU",
+    "SVN",
+    "EA19",
+    "CRI",
+]
+COUNTRY_TO_CODE_PPI = {k: v for k, v in COUNTRY_TO_CODE_GDP.items() if v in ppi_list}
+CODE_TO_COUNTRY_PPI = {v: k for k, v in COUNTRY_TO_CODE_PPI.items()}
+
+cci_list = [
+    "netherlands",
+    "switzerland",
+    "france",
+    "poland",
+    "czech_republic",
+    "sweden",
+    "united_kingdom",
+    "united_states",
+    "hungary",
+    "portugal",
+    "denmark",
+    "spain",
+    "luxembourg",
+    "greece",
+    "slovak_republic",
+    "belgium",
+    "finland",
+    "austria",
+    "italy",
+    "ireland",
+    "slovenia",
```

Comment:
Missing some countries, as per here: https://data.oecd.org/leadind/consumer-confidence-index-cci.htm

There's also a "regional" selection that groups countries: [All, OECD, EU, Euro Area, G7, G20]